fixtures/index.js

```diff
@@ -2,7 +2,8 @@ import * as agSecurity from '@aerogear/security';
 import * as agAuth from '@aerogear/auth';
 import * as agApp from '@aerogear/app';
 import * as agSync from '@aerogear/voyager-client';
-import * as agPush from '@aerogear/push'
+import * as agPush from '@aerogear/push';
+import * as offixCache from 'offix-cache';
```

Comment:
Should this dependency be added to [`scripts/build-testing-app.sh`](https://github.com/aerogear/aerogear-integration-tests/blob/master/scripts/build-testing-app.sh#L20) as well?

Reply:
`offix-cache` is included in `voyager-client` module, so it should not be needed.